packages/vuetify/src/components/VTabs/VTabs.ts

```diff
@@ -0,0 +1,292 @@
+      }
+    },
+    isReversed (): boolean {
+      return this.$vuetify.rtl && this.vertical
+    },
+    sliderStyles (): object {
+      return {
+        height: convertToUnit(this.slider.height),
+        left: this.isReversed ? undefined : convertToUnit(this.slider.left),
+        right: this.isReversed ? convertToUnit(this.slider.right) : undefined,
+        top: this.vertical ? convertToUnit(this.slider.top) : undefined,
+        transition: this.slider.left != null ? null : 'none',
+        width: convertToUnit(this.slider.width)
+      }
+    }
+  },
+
+  watch: {
+    alignWithTitle: 'callSlider',
+    centered: 'callSlider',
+    fixedTabs: 'callSlider',
+    grow: 'callSlider',
+    right: 'callSlider',
+    showArrows: 'callSlider',
+    vertical: 'callSlider',
+    '$vuetify.application.left': 'onResize',
+    '$vuetify.application.right': 'onResize',
+    '$vuetify.rtl': 'onResize'
+  },
+
+  mounted () {
+    this.$nextTick(() => {
+      window.setTimeout(this.callSlider, 30)
+    })
+  },
+
+  methods: {
+    callSlider () {
+      if (
+        this.hideSlider ||
+        !this.$refs.items ||
+        !this.$refs.items.selectedItems.length
+      ) {
+        this.slider.width = 0
+        return false
+      }
+
+      this.$nextTick(() => {
+        // Give screen time to paint
+        const activeTab = this.$refs.items.selectedItems[0]
+        /* istanbul ignore if */
+        if (!activeTab || !activeTab.$el) {
+          this.slider.width = 0
+          this.slider.left = 0
+          return
+        }
+        const el = activeTab.$el as HTMLElement
+
+        this.slider = {
+          height: this.vertical ? el.offsetHeight : 2,
+          left: this.vertical ? 0 : el.offsetLeft,
+          right: this.vertical ? 0 : el.offsetLeft + el.offsetWidth,
+          top: el.offsetTop,
+          width: this.vertical ? 2 : el.scrollWidth
+        }
+      })
+
+      return true
+    },
+    genBar (items: VNode[], slider: VNode[]) {
+      return this.$createElement(VTabsBar, this.setTextColor(this.color, {
+        staticClass: this.backgroundColor,
+        style: {
+          height: this.height ? {
+            height: convertToUnit(this.height)
+          } : null
+        },
+        props: {
+          activeClass: this.activeClass,
+          // TODO: deprecate name
+          appendIcon: this.nextIcon,
+          dark: this.dark,
+          light: this.light,
+          // TODO: deprecate name
+          prependIcon: this.prevIcon,
+          mandatory: !this.optional,
+          mobileBreakPoint: this.mobileBreakPoint,
+          showArrows: this.showArrows,
+          value: this.internalValue
+        },
+        on: {
+          'call:slider': this.callSlider,
+          change: (val: any) => {
+            this.internalValue = val
+          }
+        },
+        ref: 'items'
+      }), [
+        this.genSlider(slider),
+        items
+      ])
+    },
+    genItems (items: VNode[], item: VNode[]) {
```

Comment:
This was a confusing method and list or arguments. Perhaps a comment is warranted, or better argument names?